# Встроенные ошибки в Python

## Цель урока

После завершения этого урока вы сможете:

- Понимать различные типы встроенных ошибок в Python.
- Разбираться в их назначении и причинах возникновения.
- Использовать функцию `raise` для создания собственных ошибок.

### Встроенные ошибки
Python предоставляет множество встроенных ошибок (исключений), которые помогают вам легко обнаруживать и обрабатывать различные проблемы во время выполнения программы.  
Каждая ошибка имеет свое предназначение и может возникать в определенных ситуациях.

Вот несколько примеров встроенных ошибок в Python:
- `SyntaxError`: возникает, когда программа содержит синтаксическую ошибку, то есть некорректное использование языка Python.
- `NameError`: возникает, когда используется неопределенное имя переменной или функции.
- `TypeError`: возникает, когда операция применяется к объекту неправильного типа.
- `ZeroDivisionError`: возникает, когда деление на ноль выполняется в программе.
- `IndexError`: возникает, когда индекс выходит за пределы допустимого диапазона.

### Вызов ошибки
Чтобы создать собственную ошибку, вы можете использовать ключевое слово `raise` вместе с соответствующим типом ошибки.  
Например, чтобы вызвать ошибку `AssertionError`, вы можете использовать следующий код:

```python
raise AssertionError("Текст ошибки")
```

Когда эта строка выполняется, программа вызывает ошибку `AssertionError` с указанным текстом.  
Вызов ошибки может быть полезен, если вы хотите явно указать на возникновение ошибки в определенном месте кода.

## Задание

Вызовите ошибку `AssertionError` с любым сообщением и изучите вывод сообщений об ошибке.
